tests/pipeline/test_filter.py

```diff
@@ -395,6 +395,36 @@ def test_isfinite(self):
         )
         check_arrays(results['isfinite'], isfinite(data))
 
+    def test_smoothing_filter(self):
+        from zipline.pipeline.filters import SmoothingFilter
+
+        data = full(self.default_shape, True, dtype=bool)
+        # one column all false
```

Comment:
A good test-case for this is `~np.eye(self.default_shape, dtype=bool)`, which gives you `True` values everywhere except the diagonal.